packages/now-cli/src/util/client.ts

```diff
@@ -80,20 +72,21 @@ export default class Client extends EventEmitter {
       Object.assign(opts, {
         body: JSON.stringify(opts.body),
         headers: Object.assign({}, opts.headers, {
-          'Content-Type': 'application/json'
-        })
+          'Content-Type': 'application/json',
+        }),
       });
     }
 
     opts.headers = opts.headers || {};
     opts.headers.authorization = `Bearer ${this._token}`;
```

Comment:
```suggestion
    opts.headers.Authorization = `Bearer ${this._token}`;
```

Does this one need to be capitalized too?

Reply:
Good catch. I'll capitalize these across codebase for consistency 👍